csv/csv.go

```diff
@@ -100,14 +110,17 @@ func NewIterator(reader io.Reader, unmarshaller RecordUnmarshaller) *Iterator {
 	}
 }
 
+// NewColumnIterator creates a new iterator for column data
```

Comment:
This, and a couple of nearby comments, lack terminal `.`, but I think the convention is for complete sentences including punctuation.

Reply:
Thanks, updated.